components/collection/drop/GenerativeLayout.vue

```diff
@@ -52,20 +52,22 @@
 
       <CollectionUnlockableItemInfo :collection-id="drop?.collection" />
 
-      <hr class="my-20" />
-
-      <LazyCollectionDropItemsGrid
-        v-if="drop?.collection"
-        class="mb-14"
-        :collection-id="drop?.collection" />
+      <hr ref="divider" class="my-20" />
+
+      <LoadLazly :target="divider">
```

Comment:
Is it necessary to pass the `divider` here?

Reply:
>Is it necessary ? 

no, but I do want the divider to be the target in this case , this will make the data load a little earlier